releasenotes/notes/Bump-gosnmp-to-v1.34.1-9d5fda248a1cbba9.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Bump gosnmp to v1.34.1-0.20220306115220-ca8397b73095
+    Fixes incomplete support of SNMP v3 INFORMs
```

```suggestion
    Bump gosnmp to fix incomplete support of SNMP v3 INFORMs
```

🥜 I think including `v1.34.1-0.20220306115220-ca8397b73095` seems a bit too technical for the changelog.md file. User can still open the original PR if they need details.